docs/openapi.md

```diff
@@ -56,4 +56,63 @@ Check [swagger](https://swagger.io/) or [other tools](https://github.com/OAI/Ope
 
 ### Troubleshooting
 
-Are you sure you added `-k` parameter to curl command?
\ No newline at end of file
+* SSL certificate problem
+
+Are you sure you added `-k` parameter to curl command?
+
+* Unauthorized
+
+Make sure you clicked Authorize button, gave username and password and clicked Authorize
+
+## How to generate Python client and use it?
+
+### Prerequisites
+
+* Docker running
+* Python installed
+* Splunk with your add-on installed
+
+### Instruction
+
+1. Create directory structure and open the `tmp` directory (run in terminal: `mkdir -p tmp/restapi_client ; mkdir -p tmp/generator ; cd tmp`)
+2. Save your openapi.json file to the directory
+3. Download the rest.mustache file (`wget https://raw.githubusercontent.com/swagger-api/swagger-codegen/master/modules/swagger-codegen/src/main/resources/python/rest.mustache`)
+4. Splunk does not expect body for DELETE requests, so we need to revert modifications done for https://github.com/swagger-api/swagger-codegen/issues/9558 (`sed "s/request_body[[:blank:]]=[[:blank:]]\'{}\'/request_body = None/g" rest.mustache > generator/rest.mustache`).
+If you want to understand exactly which line of rest.mustache is affected: https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/resources/python/rest.mustache#L150
+5. Create client (`docker run --rm -v ${PWD}:/local swaggerapi/swagger-codegen-cli-v3 generate -i /local/openapi.json -l python -o /local/restapi_client -t /local/generator/`); it should appear in `restapi_client` directory
```

Comment:
Do we really need to have Docker up and running to generate the client code? Can we do this in pure Python?